templates/zerver/help/reading-strategies.md

```diff
@@ -37,9 +37,10 @@ on.
 
 * Click on a topic to narrow to messages from that topic.
 
-You can also get to a stream with `q` (stream search), by clicking on the
-magnifying glass next to `STREAMS`, or by using the autocomplete in
-[search](/help/search-for-messages).
+You can also get to a stream by: using the [keyboard
+shortcut](/help/keyboard-shortcuts) `q`, clicking on
+**STREAMS** in the left sidebar, or using the autocomplete
+in [search](/help/search-for-messages).
```

Comment:
How about:

> You can also search for a stream by clicking on **STREAMS** in the left sidebar, or using the `q` [keyboard
shortcut](/help/keyboard-shortcuts).

I don't think we need to explain the search box here.

Reply:
Applied before merging.